content/blog/announcing-keynote-speaker-for-pycon-india-2017.md

```diff
@@ -0,0 +1,31 @@
+Title: Announcing Keynote Speakers for PyCon India 2017 - Peter Wang
+Date: 2017-08-28 09:15
+Category: 2017
+Slug: announcing-keynote-speakers-for-pycon-india-2017
+Summary: Announcing PyCon India 2017 Keynote Speakers
+
+Welcoming **Peter Wang** as a Keynote Speaker for PyCon India 2017.
+
+###Peter Wang
+
+![Peter Wang](https://in.pycon.org/2017/images/speakers/peter_wang.jpg)
```

Comment:
This picture is too large. Here's a fix.
- Resize it to 500 by 500
- Upload the image to /themes/inpycon2017/static/images folder
- Link the image using https://in.pycon.org/blog/theme/images/